rmltraintfbbox/setup.py

```diff
@@ -13,7 +13,7 @@
 
 # determine GPU or CPU install via env variable
 gpu = os.getenv('RML_GPU')
-tensorflow_pkg = 'tensorflow==1.14.0' if not gpu else 'tensorflow-gpu==1.14.0'
+tensorflow_pkg = 'tensorflow==2.3.0' if not gpu else 'tensorflow-gpu==2.3.0'
```

Comment:
Is this still needed? Iirc tensorflow >=2 will include gpu stuff.

Reply:
Yup good catch. Removed gpu version.